models/labels/addresses/infrastructure/identifier/burn_addresses/labels_burn_addresses.sql

```diff
@@ -1,45 +1,44 @@
-    , ('ethereum', '0x8888888888888888888888888888888888888888', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x9999999999999999999999999999999999999999', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0xffffffffffffffffffffffffffffffffffffffff', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x000000000000000000000000000000000000dead', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x00000000000000000000045261d4ee77acdb3286', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0xdead000000000000000042069420694206942069', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x0123456789012345678901234567890123456789', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x1234567890123456789012345678901234567890', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
+    ('ethereum', 0x0000000000000000000000000000000000000000, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , now(), 'burn_addresses', 'identifier')
+    , ('ethereum', 0x0000000000000000000000000000000000000001, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , now(), 'burn_addresses', 'identifier')
+    , ('ethereum', 0x0000000000000000000000000000000000000002, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , now(), 'burn_addresses', 'identifier')
+    , ('ethereum', 0x0000000000000000000000000000000000000003, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , now(), 'burn_addresses', 'identifier')
+    , ('ethereum', 0x0000000000000000000000000000000000000004, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , now(), 'burn_addresses', 'identifier')
+    , ('ethereum', 0x0000000000000000000000000000000000000005, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , now(), 'burn_addresses', 'identifier')
```

Comment:
This use of `now()` seems a bit unsafe. We should consider hard coding a timestamp at the top of the model. E.g.,
```
{% set updated_at = "timestamp '20230-03-26'"%}
...
, ('ethereum', 0x0000000000000000000000000000000000000008, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , {{updated_at}}, 'burn_addresses', 'identifier')
```

Reply:
The `updated_at` shows the latest update/refresh time, I will put 2023-07-11 in the jinja variable.

Reply:
i think `now()` is fine, we use it in a variety of ways and it's in the docs as a timestamp(3) output
https://dune.com/docs/query/DuneSQL-reference/Functions-and-operators/datetime/#now